src/drivers/uavcan/sensors/mag.cpp

```diff
@@ -85,26 +84,6 @@ UavcanMagnetometerBridge::ioctl(struct file *filp, int cmd, unsigned long arg)
 {
 	switch (cmd) {
 
-	case MAGIOCSSCALE: {
```

Comment:
I have not verified this, but the removal of these bits might lead to a failing calibration (unless the base class does this already, which I haven't verified).

Reply:
These changes are from #14345; the new method uses multiple instances of the PX4Magnetometer class so that more than one compass can be calibrated (the calibration is done using this ioctl calls on /dev/mag<x>, so the original UavcanMagnetometerBridge could only handle one compass).

Reply:
In short, the UavcanMagnetometerBridge class itself no longer owns any /dev/magx instances, and instead creates PX4Magnetometer (and PX4Barometer) instances to handle that.